ultrasonic/build.gradle

```diff
@@ -14,10 +14,11 @@ android {
 
         minSdkVersion versions.minSdk
         targetSdkVersion versions.targetSdk
-        resConfigs 'cs', 'de', 'en', 'es', 'fr', 'hu', 'it', 'nl', 'pl', 'pt', 'pt-rBR', 'ru', 'zh-rCN', 'zh-rTW'
 
     }
 
+    bundle.language.enableSplit = false
+
```

Comment:
Otherwise Play Store might deliver a locallized stripped version, which will break the dynamic language chooser.
The additional languages are not more than a few thousand bytes anyway